tests/core/full_node/test_mempool_performance.py

```diff
@@ -69,12 +92,24 @@ async def test_mempool_update_performance(self, bt, wallet_nodes_mempool_perf, d
         blocks = bt.get_consecutive_blocks(3, blocks)
         await full_node_api_1.full_node.respond_block(full_node_protocol.RespondBlock(blocks[-3]))
 
+        fast_durations: List[float] = []
+        slow_durations: List[float] = []
+
         for idx, block in enumerate(blocks):
             start_t_2 = time.time()
```

Comment:
Do we specifically want to be measuring wall-clock time as opposed to maybe process time or thread time?  I haven't dug into the code.